examples/md-flexible/src/parsing/CLIParser.cpp

```diff
@@ -521,7 +536,7 @@ MDFlexParser::exitCodes MDFlexParser::CLIParser::parseInput(int argc, char **arg
 
   // only create objects if nothing was set by a yaml file and there was no checkpoint
```

Comment:
scenarios should only be specified through either a yaml file or cli input. However, there is no fancy error handling in case mixed input occurs. In that case only the yaml input is considered, but the output at the start of the simulation shows this.